benchmarks/dynamo/common.py

```diff
@@ -760,6 +760,39 @@ def onnxrt_model_iter_fn(model, inputs, collect_outputs=True):
 
         return onnxrt_model_iter_fn
 
+    def timed_onnx(model, onnx_model: OnnxModelFromTorchScript, inputs):
+        if current_device == "cpu" or onnx_model.is_cpu():
+            onnxrt_model_iter_fn = create_onnx_fn(onnx_model, inputs)
+        else:
+            onnxrt_model_iter_fn = create_onnx_input_binded_fn(
+                onnx_model, inputs, expected_output
+            )
+        return timed(
+            model,
+            onnxrt_model_iter_fn,
+            inputs,
+            return_result=True,
+            times=times,
+            collect_outputs=args.collect_outputs,
+        )
+
+    # Insert ONNX warm-up
+    inputs = (
+        randomize_input(copy.deepcopy(example_inputs))
+        if should_randomize_input
+        else example_inputs
+    )
```

Comment:
should randomization be inside the loop? Otherwise, the same input will be used for the 3 warmup runs